location.md

```diff
@@ -4,17 +4,61 @@ title: "All roads lead to Rome"
 permalink: /location/
 ---
 
-<div class="backdrop">
+<div class="backdrop" style="background: url(/assets/rome/the-city.jpg) 50% 40%; background-repeat:no-repeat; padding: 0; margin-bottom: 1em;" >
   <div class="popout">
     <section>
       <h1>Welcome to {{site.location.city}}</h1>
-      <p>We are very pleased to host the fourth edition of RustFest in Rome, Italy.</p>
+      <p> We are very pleased to host the fourth edition of RustFest in Rome, Italy. </p>
     </section>
   </div>
 </div>
 
-<section>
-<p>
-Rome is the capital of Italy.
-</p>
+<section markdown="1">
 
+Rome, the eternal city, is the capital of Italy with 3 million of residents. It's located in the central-western portion of the Italian Peninsula, within Lazio, along the shores of the Tiber. 
+Rome's history spans 28 centuries, leaving behind a huge number of monuments spanning from the famous Colosseum, to Piazza Di Spagna and Fontana Di Trevi.
+Also the Vatican City is an independent country inside the city boundaries of Rome, the only existing example of a country within a city: for this reason Rome has been often defined as capital of two states.  
+
+
+</section>
+
+<section class="img-grid">
+    <img src="/assets/rome/colosseum.jpg" alt="The Colosseum Photo">
+</section>
+<br/>
+<section markdown="1">
+    
+## Travelling to Rome    
+  
+### By Plane
+Rome has two Airports the [Leonardo Da Vinci](https://www.adr.it/fiumicino) airport that sit in the nearby town of Fiumicino is the biggest of Rome and allow you to travel from most locations in Europe with flights both low cost and not, it has a direct train to the main station of Rome called [Roma Termini](https://www.romatermini.com/en/) that is walking distance from the conference venue.
```

Comment:
Slightly rephrased:

Rome has two airports: the [Leonardo Da Vinci](https://www.adr.it/fiumicino) airport, located in the nearby town of Fiumicino. It is the bigger of the two airports and allows you to travel from most locations in Europe, with flights both low cost and not. It has a direct train to the main station of Rome called [Roma Termini](https://www.romatermini.com/en/), which is in walking distance from the conference venue.